crates/rpc/src/v02/method/get_state_update.rs

```diff
@@ -124,13 +124,15 @@ mod types {
     pub struct StateDiff {
         pub storage_diffs: Vec<StorageDiff>,
         #[serde_as(as = "Vec<RpcFelt>")]
-        pub declared_contract_hashes: Vec<ClassHash>,
+        pub deprecated_declared_contract_hashes: Vec<ClassHash>,
+        pub declared_contract_hashes: Vec<DeclaredClass>,
```

Comment:
Hmm, isn't this breaking our v02 JSON-RPC API?

Reply:
It's the same situation as declare v2, either 0.2.1 is updated to 0.2.2 to accommodate 0.11 changes or we have to divert from the spec to be able to handle latest state.

Reply:
Not quite the same: this changes the output even for _old_ state updates, making it an incompatible change.

Can't we just keep the old behaviour and _not_ do the rename and not add the new field?